s3_website.yml

```diff
@@ -43,7 +43,6 @@ cloudfront_distribution_config:
 # concurrency_level: 5
 
 redirects:
-  articles/can-multi-year-ssl-certificate: /articles/can-multi-year-ssl-certificates/
   articles/secondary-dns-support: /articles/secondary-dns/
```

Comment:
It appears links without the trailing slash redirect properly:

https://support.dnsimple.com/articles/can-multi-year-ssl-certificate

=>

https://support.dnsimple.com/articles/can-multi-year-ssl-certificates/